classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -4300,55 +4303,53 @@ public function refund( &$order, $transaction_id = null ) {
 			}
 		}
 
-		//get the charge
+		$success = false;
+
+		//attempt refund
 		try {
-			$charge = Stripe_Charge::retrieve( $transaction_id );
-		} catch ( \Throwable $e ) {
-			$charge = false;
-		} catch ( \Exception $e ) {
-			$charge = false;
-		}
+			
+			$secretkey = pmpro_getOption( "stripe_secretkey" );
```

Comment:
Not a big deal but always keep an eye out on your single vs double quote usage when not intending to use it with a $var

Reply:
Fixed this